src/styles/utils/FontUtils/fontFamily/singleFontFamily/index.ts

```diff
@@ -0,0 +1,57 @@
+// eslint-disable-next-line no-restricted-imports
+import type FontFamilyStyles from '@styles/utils/FontUtils/fontFamily/types';
+// eslint-disable-next-line no-restricted-imports
+import fontWeight from '@styles/utils/FontUtils/fontWeight';
+
+const fontFamily: FontFamilyStyles = {
+    SYSTEM: {
+        fontFamily: 'System',
+        fontStyle: 'normal',
+        fontWeight: fontWeight.normal,
+    },
+
+    MONOSPACE: {
+        fontFamily: 'ExpensifyMono-Regular',
+        fontStyle: 'normal',
+        fontWeight: fontWeight.normal,
+    },
+    MONOSPACE_BOLD: {
+        fontFamily: 'ExpensifyMono-Bold',
```

Comment:
Sorry for the naïve question, but why is the fontFamily `ExpensifyMono-Bold` on web and `ExpensifyMono` on native?

Reply:
You was right, I changed web font styles to use the same naming convention as native, thanks!